playbooks/ansible/roles/samba.install/tasks/main.yml

```diff
@@ -2,3 +2,7 @@
 - name: Install samba and ctdb from rpm repositories
   when: config.install.samba.repo is defined
   include_tasks: repo/main.yml
+
+- name: Install samba and ctdb from git sources
```

Comment:
Dynamic config generation using _config.yml.j2_ have a preference to git over repo in case both are defined. Thus in such a situation we end up having only `config.install.samba.git` in final _config.yml_.

Reply:
ok. I can see that in config.yml.j2. Thanks.